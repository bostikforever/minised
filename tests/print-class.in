	az
Bz
acz
aZz
a4z
a:z
a?z
a!z
a#z
a$z
a%z
a&z
a*z
a(z
a)z
a-z
a,z
a.z
a/z
