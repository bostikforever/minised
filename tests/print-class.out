	xx
xx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
xxx
